### PR TITLE
More Readme and failures cause non-zero exits

### PR DESCRIPTION
**Pull request by ajohanssonwalder**

### CODE REVIEW

File: modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java [hunk 1]
```diff
@@ -47,6 +48,7 @@ private static void printResults(List<StyleError> errorList) {
             for (StyleError error : errorList) {
                 System.out.println(error.toString());
             }
+            System.exit(1);
```
Comment (JFCote) [inline modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java]: I'm not sure about this. The program is successful, but it has found some errors... Was there a reason for you to add this?

File: modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java [hunk 2]
```diff
@@ -37,6 +37,7 @@ public static void main(String[] args) {
             printResults(openApiSpecStyleValidator.validate(parameters));
         } catch (ParseException e) {
             System.out.println("There was something wrong in your request. Please check documentation for more info");
+            System.exit(1);
```
Comment (JFCote) [inline modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java]: Very nice addition.

File: README.md [hunk 3]
```diff
@@ -55,20 +55,52 @@ The project is configured to use gradle. To build the jar, just do:
...
+This IDE is cross platform so it should work on any OS. This project was created using this tool and you can just open
 it directly. Maybe you can use other tools to open the project but I will not provide any support.
 
 ### To launch
 
 `java -jar swagger-style-validator.jar -s ./path/to/spec.yaml -o ./path/to/options.json`
 
--s|-source : The path to your yaml spec file
--o|-options : (Optional) The path to your json options file 
+or example
```
Comment (JFCote) [inline README.md]: maybe change this to: Another example using the default output path of the jar file

File: README.md [hunk 4]
```diff
@@ -55,20 +55,52 @@ The project is configured to use gradle. To build the jar, just do:
...
+|Parameter|Required?|Description|
+|---|---|---|
+|-s, -source|yes|The path to your json/yaml spec file|
+|-o, -options|no|The path to your json options file|
+
+#### Options File
+The options file is described in json (example in `specs/options.json`), and has the following possible values:
 
-Please check the `options.json` file for example on how to provide options to the tool.
+|Option|Type|Possible Values|
```
Comment (JFCote) [inline README.md]: very nice addition! Thanks!

File: modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java (hunk 1 above)
Comment (ajohanssonwalder) [reply modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java]: You could be right. The reason I added this is to make it easier to use during CI tests. If the style validation fails, I want the test to exit with a non-zero so it stops the pipeline. Thoughts?

Comment (JFCote) [reply modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java]: It makes sense. You can keep this code. I will need to check with other project what is the "standard" way of dealing with this in a validator.

**Comment by JFCote** [comment]
@ajohanssonwalder Merged! Thanks for the PR.
By the way, next time you contribute, please create a branch in your fork to follow good Github practice. See this for more info: https://gist.github.com/Chaser324/ce0505fbed06b947d962

**Comment by ajohanssonwalder** [comment]
hehe thanks Jean-Francois! 
I normally do feature branches, but I have never forked or worked on public repos before 😄  😂   Will keep that in mind for future works

**Pull request merged**